instrumentation/faraday/lib/opentelemetry/instrumentation/faraday/instrumentation.rb

```diff
@@ -28,6 +28,7 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
 
         option :span_kind, default: :client, validate: %i[client internal]
```

Comment:
What about adding an option here like `:none`?

Reply:
Could you elaborate on this? The current behavior of setting the flag to false will ensure no spans are suppressed.